src/pages/api/event/[id]/repairers.ts

```diff
@@ -0,0 +1,27 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import apiHandler from "@/lib/api-handler";
+import { EventRepairer } from "@/types";
+
+import prisma from "../../../../lib/prisma";
+
+export default apiHandler({
+  post: createRepairer
+});
+
+async function createRepairer(
+  req: NextApiRequest,
+  res: NextApiResponse<EventRepairer>
+) {
+  const { userId } = req.body;
+  const { id } = req.query;
```

Comment:
Consider validating that an event exists first as well, otherwise throw a `throw new ApiError(HttpStatusCode.NotFound, "Event not found");`